This patch silences a spurious warning that occurs when running the GUI under VirtualGL
with EGL enabled.
---
--- VTK/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx.orig        2023-08-23 16:35:23.418470811 +0200
+++ VTK/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx     2023-08-23 16:36:10.830654656 +0200
@@ -471,8 +471,8 @@
     this->GlewInitValid = (result == GLEW_OK);
     if (!this->GlewInitValid)
     {
-      const char* errorMsg = reinterpret_cast<const char*>(glewGetErrorString(result));
-      vtkErrorMacro("GLEW could not be initialized: " << errorMsg);
+      //const char* errorMsg = reinterpret_cast<const char*>(glewGetErrorString(result));
+      //vtkErrorMacro("GLEW could not be initialized: " << errorMsg);
       return;
     }
